cli/commitflags.go

```diff
@@ -1,4 +1,4 @@
-package lake
+package cli
```

Comment:
Maybe change ZED_LAKE_USER to ZED_USER below? And there's a "zed lake" in the CommitFlags comment.